website/docs/r/cloudwatch_log_transformer.html.markdown

```diff
@@ -0,0 +1,321 @@
+---
+subcategory: "CloudWatch Logs"
+layout: "aws"
+page_title: "AWS: aws_cloudwatch_log_transformer"
+description: |-
+  Terraform resource for managing an AWS CloudWatch Logs Transformer.
+---
+
+# Resource: aws_cloudwatch_log_transformer
+
+Terraform resource for managing an AWS CloudWatch Logs Transformer.
+
+## Example Usage
+
+### Basic Usage
+
+```terraform
+resource "aws_cloudwatch_log_group" "example" {
+  name = "example"
+}
+
+resource "aws_cloudwatch_log_transformer" "example" {
+  log_group_identifier = aws_cloudwatch_log_group.example.name
+  transformer_config {
+    parse_json {}
+  }
+}
```

Comment:
The resource type being documented should be at the top of the example configuration